metricbeat/module/aerospike/_meta/config.reference.yml

```diff
@@ -3,3 +3,6 @@
   enabled: true
   period: 10s
   hosts: ["localhost:3000"]
+
+  # Set to true to publish fields with null values in events.
+  #keep_null: false
```

Comment:
I'd prefer not to add this to the reference config of all modules, we don't do it with other common options like `fields`, `service.name` in metricbeat or `pipeline` in filebeat.

Reply:
Done in de6bb5d724212d844919b514480ab5cc2d732c32.